.github/actions/analyze-test-runs/action.yml

```diff
@@ -33,6 +33,7 @@ runs:
     env:
       BUILD_OUTPUT_FILE_PATH: ${{ inputs.buildOutputFilePath }}
     run: |
+      set -eoux
```

Comment:
We can discuss whether we want to keep it, but I find it quite useful to better trace what is actually failing.


https://explainshell.com/explain?cmd=set+-euox

Reply:
I think, it is useful since it will print on error cases, it shouldn't affect the successful build time as well.